.github/workflows/periodic.yml

```diff
@@ -163,43 +163,6 @@ jobs:
       docker-image: ${{ needs.linux-bionic-cuda11_7-py3_10-gcc7-debug-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-bionic-cuda11_7-py3_10-gcc7-debug-build.outputs.test-matrix }}
 
-  linux-bionic-cuda11_8-py3_8-gcc7-debug-build:
```

Comment:
I'd be more inclined to continue testing the latest 11.8 cuda instead of the older 11.7 cuda.

@malfet, thoughts?

Reply:
Sound good!  Let's keep 11.8 here as I think the consensus is to move toward that version (with CUDA 12 coming soon)